src/components/homesections/homesections.js

```diff
@@ -428,25 +428,28 @@ import ServerConnections from '../ServerConnections';
         };
     }
 
-    function getItemsToResumeHtml(items) {
-        const cardLayout = false;
-        return cardBuilder.getCardsHtml({
-            items: items,
-            preferThumb: true,
-            defaultShape: getThumbShape(),
-            overlayText: false,
-            showTitle: true,
-            showParentTitle: true,
-            lazy: true,
-            showDetailsMenu: true,
-            overlayPlayButton: true,
-            context: 'home',
-            centerText: !cardLayout,
-            allowBottomPadding: false,
-            cardLayout: cardLayout,
-            showYear: true,
-            lines: 2
-        });
+    function getItemsToResumeHtmlFn(useEpisodeImages) {
+        return function (items) {
+            const cardLayout = false;
+            return cardBuilder.getCardsHtml({
+                items: items,
+                preferThumb: true,
+                inheritThumb: !useEpisodeImages,
+                defaultShape: getThumbShape(),
+                overlayText: false,
+                showTitle: true,
+                showParentTitle: true,
+                lazy: true,
+                showDetailsMenu: true,
+                overlayPlayButton: true,
+                context: 'home',
+                centerText: !cardLayout,
+                allowBottomPadding: false,
+                cardLayout: cardLayout,
+                showYear: true,
+                lines: 2
+            });
+        }
```

Comment:
Looks like there is a lint error here now due to a missing semicolon.

Reply:
Fixed. Apparently I've overlooked the mail from Github Actions about that problem. Everything should be fine right now.

Reply:
That is easy to do. :sweat_smile: